src/Resources/translations/validators.hu.xlf

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>This value is not a valid phone number.</source>
+                <target>Hibás telefonszám.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>This value is not a valid fixed-line number.</source>
+                <target>Hibás vonalas telefonszám.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>This value is not a valid mobile number.</source>
+                <target>Hibás mobil szám.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>This value is not a valid pager number.</source>
+                <target>Hibás csipogó szám.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>This value is not a valid personal number.</source>
+                <target>Hibás központi telefonszám (follow me number).</target>
```

Comment:
Is it normal there is still some English here?

Reply:
I tried my best to translate this accurately, but I think there is not exact term for this in Hungarian, so I left the English term too, as a reference to the exact technical meening.

Reply:
Ok this sounds great.